docs/UserGuide.md

```diff
@@ -146,6 +146,44 @@ Examples:
 * `changeqty 1 a/+1` increases the quantity of the 1st food item by 1.
 * `changeqty 2 a/-2` decreases the quantity of the 2nd food item by 2.
 
+### Sort food items by description: `sortdesc`
+
+Sorts the list of food items by description, then by expiry date from oldest to newest, then by priority from high to low. 
+
+* Sorting by description consists of first sorting lexicographically, then by the description's first characters.
```

Comment:
Maybe we should avoid using "lexicographically"? Many users might not be familiar with this word. Would "in alphabetical order" better?

Reply:
Alphabetical order is actually different from lexicographical order, so here it is more accurate to use "lexicographically". Instead of changing the meaning, maybe including a glossary to explain this is better.